app/controllers/pensions/transferIntoOverseasPensions/TransferIntoOverseasPensionsCYAController.scala

```diff
@@ -73,7 +73,7 @@ class TransferIntoOverseasPensionsCYAController @Inject()(authAction: Authorised
   }
 
 
-  def submit(taxYear: Int): Action[AnyContent] = authAction.async { implicit request =>
+  def submit(taxYear: Int): Action[AnyContent] = auditProvider.transfersIntoOverseasPensionsUpdateAuditing(taxYear).async { implicit request =>
     pensionSessionService.getAndHandle(taxYear, request.user) { (cya, prior) =>
```

Comment:
You need to remove the pensionSessionService.getAndHandle code of line 77 etc.. as the implicit request: UserSessionDataRequest has request.pensions